helm-charts/verticadb-operator/README.md

```diff
@@ -4,6 +4,7 @@ This helm chart will install the operator and an admission controller webhook.
 |----------------|-------------|---------------|
 | image.name | The name of image that runs the operator. | vertica/verticadb-operator:1.0.0 |
 | image.repo | Repo server hosting image.name | null (DockerHub) |
+| image.pullPolicy | When to pull the image from DockerHub  | IfNotPresent |
```

Comment:
We might not be pulling from DockerHub.  I suggest this for the description:
`The pull policy for the image that runs the operator`